README.md

```diff
@@ -79,3 +79,16 @@ features = [ "alloc" ]
 7. To run the project, from its root, you should now be able to `crank run` successfully!
 
 If you want an example of an independent project following these conventions, go check out [Nine Lives](https://github.com/bravely/nine_lives).
+
+## Updating Bindings
+
+If there's a newer [Playdate SDK](https://play.date/dev/) available that updates the C API, the crankstart bindings should be updated to match.
+Here's a guide.
+
+1. Install [the dependencies for bindgen](https://rust-lang.github.io/rust-bindgen/requirements.html).
+2. Install [bindgen-cli](https://rust-lang.github.io/rust-bindgen/command-line-usage.html).
+3. On Linux, install the 32-bit glibc development package, which will be called something like `glibc-devel-32bit`.
```

Comment:
Would it be worth mentioning that you also need the gcc-arm-none-eabi toolchain? (e.g. from <https://developer.arm.com/Tools%20and%20Software/GNU%20Toolchain> or packaged for linux distro)

Reply:
Yes, thanks!  I was sure I had forgotten something but... forgot :D